### ISSUE TITLE
Implement profile UI

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#49]
### Who

- [ ] Anonymous visitor
- [x] Patient
- [ ] Doctor
- [ ] Content editor
- [ ] Marketer
- [ ] Administrator
- [ ] Developer

### Needs what

To edit their profile information.

### Why

Why is the feature needed?

### Issue expands upon

This issue expands upon another issue(s): #20 

### Blockers

This issue is blocked by the following issue(s): #`issue number`
